network-api/networkapi/templates/foundation-base.html

```diff
@@ -1,6 +1,6 @@
 <!-- This is a copy/paste of landing-page.html, modified to be useful for fellowships to start. Ideally the entire site will extend this template eventually -->
 <!DOCTYPE html>
-<html>{% load settings_value %}{% load active_nav %}
+<html>{% load settings_value %}{% load fellowship_active_nav %}
```

Comment:
I realize I'm a bit concerned with editing of this file too much to be specific to fellows (and still calling it `foundation-base`). Perhaps we should abstract out a level so we can rely on a true "foundation base" generated from the original pug file?  I'd say for this PR, let's start with renaming this file.

Reply:
k i'll rename it to `fellowships_base.html` as the first step